Dockerfile

```diff
@@ -15,5 +15,11 @@ RUN apt-get install -y python2.7 python3.6 python-pip git nodejs yarn gettext py
 COPY . /kolibri
 
 VOLUME /kolibridist/  # for mounting the whl files into other docker containers
-CMD cd /kolibri && pip install -r requirements/dev.txt && pip install -r requirements/build.txt && yarn install && make dist && cp /kolibri/dist/* /kolibridist/
+CMD cd /kolibri && pip install -r requirements/dev.txt \
+	&& pip install -r requirements/vf.txt \
+	&& pip install -r requirements/build.txt \
+	&& python -m kolibri plugin kolibri-instant-schools-plugin enable \
+	&& yarn install \
+	&& make dist REQUIREMENTS=requirements/vf.txt \
```

Comment:
In general we try to merge changes from release branches back to develop, so we'll have to be careful about this.

I wonder if we should be cherry-picking the changes from develop to avoid merge conflicts?